swagger/formats.go

```diff
@@ -0,0 +1,35 @@
+package swagger
+
+import (
+	"regexp"
+
+	"github.com/go-openapi/strfmt"
+)
+
+func InitCustomFormats() {
+	m := mongoID("")
+	strfmt.Default.Add("mongo-id", &m, isMongoID)
+}
+
+var mongoRegExp = regexp.MustCompile("^[0-9a-f]{24}$")
+
+func isMongoID(str string) bool {
+	return mongoRegExp.MatchString(str)
```

Comment:
@agclever mentioned in another PR that the regexp condition might not be sufficient to identify valid ObjectIDs. If this is true, then we should use `bson.IsObjectIdHex`.